flavour/gcloud/flavour.yaml

```
@@ -11,4 +11,5 @@ description: |
     * Mount a directory that contains the JSON file into the CloudControl container and set GCLOUD_KEYPATH accordingly
 configuration:
     - "Environment GCLOUD_PROJECTID: The id of the Google Cloud project to connect to"
+    - "Environment GCLOUD_USE_SA: Use a service account to log into Google Cloud. Requires GCLOUD_KEYPATH (defaults to false)"
```

I don't know if we have something similar elsewhere, but it would be helpful to tell how to provide the boolean value. A little hint is already given, but it is confusing, because someone could read that the KEYPATH requirement defaults to false.
Proposal:
```suggestion
    - "Environment GCLOUD_USE_SA (Possible values: true, false. Defaults to false): Use a service account to log into Google Cloud. Requires GCLOUD_KEYPATH"
```

---

Agreed and done. After these changes we should default that for all features.